### PR TITLE
fix tab behaviour

### PR DESCRIPTION
**Pull request by Diaskhan**
clear a code, 
and make tab beahavior morre integrated with DataGrid

I found a bug, tabcoomand doesnt change a focus, when we press Tab key!

There is emmbeded property IsFocused in 
https://docs.avaloniaui.net/docs/concepts/input/focus#isfocused-and-focusable

### CODE REVIEW

File: src/SmartCommander/ViewModels/MainWindowViewModel.cs [hunk 1]
```diff
@@ -39,8 +39,11 @@ public MainWindowViewModel()
             TabCommand = ReactiveCommand.Create(ChangeSelectedPane);
             OptionsCommand = ReactiveCommand.CreateFromTask(ShowOptions);
 
-            LeftFileViewModel = new FilesPaneViewModel(this) { IsSelected = true };
+            LeftFileViewModel = new FilesPaneViewModel(this);
+            LeftFileViewModel.FocusChanged += OnFocusChanged;
+
             RightFileViewModel = new FilesPaneViewModel(this);
+            LeftFileViewModel.FocusChanged += OnFocusChanged;
```
Comment (anovik) [inline src/SmartCommander/ViewModels/MainWindowViewModel.cs:46]: Not sure, why you add FocusChanged event handler for LeftFileViewModel twice? Should it be RightFileViewModel in the second case?

File: src/SmartCommander/ViewModels/MainWindowViewModel.cs [hunk 2]
```diff
@@ -152,44 +162,27 @@ public FilesPaneViewModel SecondPane
...
                 {
-                    return LeftFileViewModel;
+                    return RightFileViewModel;
                 }
-                throw new Exception(Resources.ErrorNoPane);
             }
         }
+        private FilesPaneViewModel _SelectedPane;
 
         public FilesPaneViewModel SelectedPane
```
Comment (anovik) [inline src/SmartCommander/ViewModels/MainWindowViewModel.cs:177]: Maybe turn it to auto property?

**Review by anovik — COMMENTED** [review]
Looks good generally, could you please address the couple of comments?

File: src/SmartCommander/ViewModels/MainWindowViewModel.cs (hunk 1 above)
Comment (Diaskhan) [reply src/SmartCommander/ViewModels/MainWindowViewModel.cs:46]: fixed

File: src/SmartCommander/ViewModels/MainWindowViewModel.cs (hunk 2 above)
Comment (Diaskhan) [reply src/SmartCommander/ViewModels/MainWindowViewModel.cs:177]: fixed

Comment (anovik) [reply src/SmartCommander/ViewModels/MainWindowViewModel.cs:177]: Sorry, I don't mean that actually. My idea we can turn SelectedPane to auto property and get rid of _SelectedPane.

Comment (Diaskhan) [reply src/SmartCommander/ViewModels/MainWindowViewModel.cs]: I get It. 
_SelectedPane removed

**Review by anovik — APPROVED** [review]

**Comment by anovik** [comment]
@Diaskhan Thanks so much for your PR. May I only ask you to write more meaningful messages for commits?

**Pull request merged**